mod-pubsub-server/pom.xml

```diff
@@ -65,6 +65,12 @@
       <artifactId>liquibase-core</artifactId>
       <version>3.4.1</version>
     </dependency>
+    <dependency>
+      <groupId>org.folio</groupId>
+      <artifactId>data-import-utils</artifactId>
```

Comment:
mod-pubsub should be self-independent. It should not depend on data-import, because changes in data-import-utils  may affect this module